std/internal/math/biguintcore.d

```diff
@@ -1113,6 +1113,28 @@ public:
         return .hashOf(data);
     }
 
+    /**
+    Random generate of the number of specified uintLength.
+    Params:
+        uintLength = the specified uintLength.
+    */
+    void randomGenerate(const size_t uintLength) @safe
+    {
+        assert(uintLength > 0);
+
+        import std.random : Mt19937, unpredictableSeed, uniform;
+        Mt19937 rnd;
+        rnd.seed(unpredictableSeed);
```

Comment:
Could use `std.random.rndGen` instead of seeding a new PRNG each time.

Reply:
@n8sh, thank you very much. :)